tests/test_response.py

```diff
@@ -67,3 +69,7 @@ def test_response_body_passes_through_when_base64(self):
             "statusCode": 666,
             "isBase64Encoded": True,
         }
+
+    def test_unsported_header_raises(self) -> None:
```

Comment:
typo: _"un**suppo**rted"_

Reply:
It would be good to place a full sentence too, including the reason - it will be valuable for future developers.